scripts/run_node.sh

```diff
@@ -2,4 +2,27 @@
 # This script is meant to be run on Unix/Linux based systems
 set -e
 
-exec ../target/release/madara --dev --tmp --rpc-external --execution native --pool-limit=100000 --pool-kbytes=500000 --rpc-methods=unsafe --rpc-cors=all --in-peers=0 --out-peers=1 --no-telemetry
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    --with-state-root)
+      with_state_root=true
+      ;;
+    *)
+      echo "Unknown argument: $1"
+      exit 1
+      ;;
+  esac
+  shift
+done
+
+# Check if the --with-state-root flag is provided
+if [[ $with_state_root ]]; then
+    echo "Running with state root..."
+    cargo build --release --features madara-state-root
+    exec ../target/release/madara --dev --tmp --rpc-external --execution native --pool-limit=100000 --pool-kbytes=500000 --rpc-methods=unsafe --rpc-cors=all --in-peers=0 --out-peers=1 --no-telemetry
+else
+    echo "Running without state root..."
+    cargo build --release
+    exec ../target/release/madara --dev --tmp --rpc-external --execution native --pool-limit=100000 --pool-kbytes=500000 --rpc-methods=unsafe --rpc-cors=all --in-peers=0 --out-peers=1 --no-telemetry
+fi
```

Comment:
strictly speaking this logic is not required for running benchmark with state-root enabled, because we are specifying the `cargo build --features --release madara-state-root` step in CI itself.

But i have kept it here so its easier for anyone else to use this. And also now we can run this script directly without first running `cargo build` manually